PhysicsTools/PatAlgos/python/producersLayer1/photonProducer_cfi.py

```diff
@@ -3,7 +3,7 @@
 patPhotons = cms.EDProducer("PATPhotonProducer",
     # input collection
     photonSource = cms.InputTag("gedPhotons"),
-             
+    electronSource = cms.InputTag("gedGsfElectrons"),                            
```

Comment:
Hi,
it doesn't matter since it is taking the default value from fillDescriptions

iDesc.add<edm::InputTag>("photonSource", edm::InputTag("no 
default"))->setComment("input collection");
iDesc.add<edm::InputTag>("electronSource", edm::InputTag("no 
default"))->setComment("input collection");
iDesc.add<edm::InputTag>("conversionSource", 
edm::InputTag("allConversions"))->setComment("input collection");


On 3/12/15 15:23, Tae Jeong Kim wrote:
>
> In PhysicsTools/PatAlgos/python/producersLayer1/photonProducer_cfi.py 
> <https://github.com/cms-sw/cmssw/pull/7946#discussion_r26304827>:
>
> > @@ -3,7 +3,7 @@
> >  patPhotons = cms.EDProducer("PATPhotonProducer",
> >      # input collection
> >      photonSource = cms.InputTag("gedPhotons"),
> > -
> > +    electronSource = cms.InputTag("gedGsfElectrons"),
>
> @matteosan1 <https://github.com/matteosan1> Shouldn't we add 
> "conversionSource" as well here?
>
> Taejeong
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-sw/cmssw/pull/7946/files#r26304827>.
>

+++++++++++
Matteo Sani
University of California, San Diego, US.
Tel: +41227671577
CERN Office: 40-3-A02